ddl/backfilling.go

```diff
@@ -273,7 +273,7 @@ func (w *backfillWorker) handleBackfillTask(d *ddlCtx, task *reorgBackfillTask,
 		rc.increaseRowCount(int64(taskCtx.addedCount))
 		rc.mergeWarnings(taskCtx.warnings, taskCtx.warningsCount)
 
-		if num := result.scanCount - lastLogCount; num >= 30000 {
+		if num := result.scanCount - lastLogCount; num >= 90000 {
```

Comment:
For 90000 row print log, it will be how much of interval of time?

Reply:
Normally, throughput of adding index using ingest is about 200k rows per second.

Reply:
how about we set 200k rows print one log?

Reply:
Maybe 200k is too large for the old implementation, the average throughput is 20k rows per second.

Reply:
I think we can create a sliding window to static the throughput. so we can adapt the rows for machine performance.

Lucky, I have create a sliding windows at[util/window](https://github.com/pingcap/tidb/pull/37388/files#diff-d81a95b846275ac48ebf928efa5612e0ad44d16d291b3dd5586480da7034e01e) at this [pr](https://github.com/pingcap/tidb/pull/37388/files#diff-d81a95b846275ac48ebf928efa5612e0ad44d16d291b3dd5586480da7034e01e)